nbs/010_engine.ipynb

```diff
@@ -40,7 +40,9 @@
     "from abc import ABC, abstractmethod\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #251.            query_graph, root_node = self.plan_query(q,rewrites)
</pre><p>why dont you allow defining output csv here as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spannerlib/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>